### PR TITLE
#1589 Added my name to credits

### LINKED ISSUE #1589: Yash Jayaprakash - Credits
### Description

Giving myself credit for the amazing developing work I will be doing at NER!

### Acceptance Criteria

See my name in my favorite color on the credits page.

### Proposed Solution

Add my name and favorite color to the "names" array in the CreditsPage.tsx file

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by yashjay1**
## Changes

Added name to the Credits Page.

## Screenshots

Screenshot 2023-10-03 at 11 35 08 PM
Screenshot 2023-10-03 at 11 33 47 PM

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #1589

### CODE REVIEW

File: src/backend/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "concat-stream": "^2.0.0",
     "cookie-parser": "^1.4.5",
     "cors": "^2.8.5",
+    "docker": "^1.0.0",
```
Comment (RChandler234) [inline src/backend/package.json:19]: you shouldn't have changes to any of these files, you should only be committing changes to the credits page. You can fix this by deleting the package-lock.json, comitting that change, and then deleting line 19 that you added in the package.json, running yarn install and then committing those changes

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Review by yashjay1 — COMMENTED** [review]
@RChandler234 made the changes here

**Review by RChandler234 — CHANGES_REQUESTED** [review]
you still need to do the second part where you delete line 19, run yarn install again, and then commit the code!

**Comment by yashjay1** [comment]
Weird, I thought I did that. Anyways made the changes now :D

**Comment by yashjay1** [comment]
Weird, I thought I did. Anyways made the changes! :D

**Review by RChandler234 — CHANGES_REQUESTED** [review]
make sure to remove the yarn.lock changes

**Pull request closed without merging**